gzip.go

```diff
@@ -131,6 +133,18 @@ func (w *GzipResponseWriter) Flush() {
 	}
 }
 
+// Hijack implements http.Hijacker. If the underlying ResponseWriter is a
+// Hijacker, its Hijack method is returned. Otherwise an error is returned.
+func (w *GzipResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	if hj, ok := w.ResponseWriter.(http.Hijacker); ok {
+		return hj.Hijack()
+	}
+	return nil, nil, fmt.Errorf("Hijacker interface is not supported")
```

Comment:
Just one _tiny_ thing: `s/H/h/` on that error string, please.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings